src/controllers/update/update.review.beneficial.owner.gov.ts

```diff
@@ -0,0 +1,54 @@
+import { UPDATE_BENEFICIAL_OWNER_BO_MO_REVIEW_URL, UPDATE_BENEFICIAL_OWNER_TYPE_URL, UPDATE_REVIEW_BENEFICIAL_OWNER_GOV_PAGE } from "../../config";
+import { NextFunction, Request, Response } from "express";
+import { getApplicationData, removeFromApplicationData, setApplicationData } from "../../utils/application.data";
+import { logger } from "../../utils/logger";
+import { ApplicationDataType } from "../../model";
+import { setBeneficialOwnerData } from "../../utils/beneficial.owner.gov";
+import { v4 as uuidv4 } from "uuid";
+import { BeneficialOwnerGovKey } from "../../model/beneficial.owner.gov.model";
+import { Session } from "@companieshouse/node-session-handler";
+import { saveAndContinue } from "../../utils/save.and.continue";
+
+export const get = (req: Request, res: Response) => {
+  logger.debugRequest(req, `${req.method} ${req.route.path}`);
+  const appData = getApplicationData(req.session);
+  const index = req.query.index;
+
+  let dataToReview = {};
+  if (appData?.beneficial_owners_government_or_public_authority){
+    dataToReview = appData?.beneficial_owners_government_or_public_authority[Number(index)];
+  }
```

Comment:
in my changes need to add in the below so principalAddress is visibile if user revisited the page, and so service address is viewable anytime

```principalAddress = (dataToReview) ? mapDataObjectToFields(dataToReview[PrincipalAddressKey], PrincipalAddressKeys, AddressKeys) : {};
      serviceAddress = (dataToReview) ? mapDataObjectToFields(dataToReview[ServiceAddressKey], ServiceAddressKeys, AddressKeys) : {};```